src/tests/EdFi.DataManagementService.Tests.E2E/Features/Descriptors/DescriptorsValidation.feature.disabled

```diff
@@ -0,0 +1,285 @@
+Feature: Descriptors are a set of mechanisms to support flexible enumerations or code sets
+
+        Background:
+            Given the system has this "educationOrganizationCategoryDescriptors"
+                  | codeValue | description | namespace                                               | shortDescription |
+                  | School    | School      | uri://ed-fi.org/EducationOrganizationCategoryDescriptor | School           |
+              And the system has this "disabilityDescriptors"
+                  | codeValue       | description          | namespace                            | shortDescription     |
+                  | Disability Test | User Disability Test | uri://ed-fi.org/DisabilityDescriptor | User Disability Test |
```

Comment:
Do we really need this background section?

Reply:
This might cause conflict with the DMS-266 functionality, so hopefully we don't need it this way. If you really need to create descriptors in background section it's best to use `Given the system has these descriptors`